src/desktop/components/main_layout/templates/instant-page.jade

```diff
@@ -0,0 +1,132 @@
+if sd.ENABLE_INSTANT_PAGE
+  //- Prefetch links on hover. See: https://instant.page/
+  script( type="text/javascript" ).
```

Comment:
Will this run through babel/be minified? It's got es6 stuff in it and we'd definitely want it minified.

Reply:
We no longer support browsers that don't support this code; they'll get redirected to a non supported page. I'll collapse it though.

Reply:
This comment from the author is why I didn't minify: 

> https://github.com/instantpage/instant.page/pull/22#issuecomment-467391090

> There’s actually no performance to be gained from minification because instant.page fits in a single TCP packet (under ~1.4kB).